lib/Gcis/syncer/article.pm

```diff
@@ -71,6 +91,21 @@ sub _set_journal_title {
     return $return_value;
 }
 
+sub _fix_issn {
+    my $s = shift;
+    my $issn = shift;
+    my $title = shift;
+    my %issn_map = (
+        "Eos, Transactions American Geophysical Union"    => {"0096-3941" => "2324-9250"}, 
```

Comment:
I think you may be able to use the worldcat API this --
http://xissn.worldcat.org/webservices/xid/issn/0096-3941?method=getHistory&format=xml
http://xissn.worldcat.org/xissnadmin/doc/api.htm
(the first URL finds 2324-9250)